### ISSUE TITLE
Deleting one item from group causes duplicates of all other items to appear.

### ISSUE THREAD

**Issue by kacper1112** [mattwright324/youtube-comment-suite#63]
Here I added 5 videos to the default group. After deleting one of them (does not matter which one) all of the other items appear twice in the list. This happens only once - deleting another item does not create more duplicates, the existing ones still remain though. 

Platform: Windows 10
Java: 1.8.0_271 (Oracle)

Ran both using the release 1.4.3 jar file and compiled from source myself.

![before](https://user-images.githubusercontent.com/36137483/104065411-8c9d1500-51b4-11eb-8c80-5f178016bbea.PNG)
_deleted  "Waiting for love"_
![after](https://user-images.githubusercontent.com/36137483/104065408-8b6be800-51b4-11eb-8a6d-48e99dfd73da.PNG)

Note that the item counters in buttons include the duplicated videos, so they are probably not only a visual bug.

**Comment by mattwright324** [comment]
Hey @kacper1112

I remember encountering duplicate listed items before but didn't know how to reproduce it. I see it happening now when I delete items. I'll look into this, thanks!

**Comment by mattwright324** [comment]
This seems to be caused because I am reloading twice at the exact same time.

I have this fixed in a commit to the `develop` branch 9f9e4423ae3568042959c3bbf1b7a0c431171c3b and will be in the next release.
